tasks/shared_libraries/run_tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
This script should reside in the following directory based on where you have the code.

`tasks/shared_libraries/recovery/bin`